Tasks/Task2/Summ.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Tasks.Task2
+{
+    /// <summary>
+    /// Summing
+    /// </summary>
+    public class Summ
+    {
+        /// <summary>
+        /// Summa with ignoring duplicate numbers
+        /// </summary>
+        /// <param name="numbers">
+        /// numbers
+        /// </param>
+        public static int SummIgnoringDuplicates(List<int> numbers)
+        {
+            HashSet<int> unique = new HashSet<int>();
+            HashSet<int> duplicates = new HashSet<int>();
+
+            int summ = 0;
+
+            foreach (int number in numbers)
+            {
+                if (duplicates.Contains(number))
+                {
+                    continue;
+                }
+                if (unique.Contains(number))
+                {
+                    unique.Remove(number);
+                    duplicates.Add(number);
+                }
+                else
+                {
+                    unique.Add(number);
+                }
+            }
+            foreach (int number in unique)
+            {
+                summ += number;
+            }
+            return summ;
+        }
```

Comment:
I think it can be done in diffrent way, will provide suggestion on Monday

Reply:
sum = numbers.GroupBy(x => x).Where( n => n.Count() == 1).Select(y => y.Key).Sum();
or even shorter:
sum = numbers.Where(x => numbers2.Count(n => n == x) == 1).Sum();